src/test/java/ticTacToe/grid/rulesTest.java

```diff
@@ -13,36 +13,36 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 
-public class RowsTest {
+public class rulesTest {
```

Comment:
The class name should start with a capital `R`

Reply:
Yeah, I made a mess there